core/src/main/java/com/graphhopper/routing/ch/CHAlgoFactoryDecorator.java

```diff
@@ -210,7 +212,7 @@ public CHAlgoFactoryDecorator setWeightingsAsStrings(String... weightingNames) {
 
     public List<String> getWeightingsAsStrings() {
         if (this.weightingsAsStrings.isEmpty())
-            throw new IllegalStateException("Potential bug: chWeightingList is empty");
+            throw new IllegalStateException("Potential bug: weightingsAsStrings is empty");
```

Comment:
I know :). I just stumbled over it and was wondering why we are throwing such a generic exception here.